_trash/pr845_parse.py

```
@@ -0,0 +1,16 @@
+import json
+
+c = json.load(open(r"C:\Users\Admin\AppData\Local\Temp\pr845_rc.json", encoding="utf-8"))
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)